sdk/formrecognizer/ai-form-recognizer/samples/v5/javascript/buildModel.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * This sample shows how to programmatically build a model with a single document type using a training data set.
+ *
+ * The Document Intelligence service expects the training data to be organized and labeled according to a particular
+ * convention and stored in an Azure Storage container. For more information about creating a training data set, please
+ * see the information at the following link to the service's documentation:
+ *
+ * https://aka.ms/azsdk/formrecognizer/buildtrainingset
```

Comment:
this url too?
`https://aka.ms/azsdk/documentintelligence/buildtrainingset`

Reply:
for the next release. I think if I had more time I would go ahead and rename the URLs, but don't want to go through dozens of files replacing URLs at the last minute.